processing/src/test/java/org/apache/druid/query/groupby/GroupByQueryRunnerTest.java

```diff
@@ -5843,6 +5845,29 @@ public void testDifferentIntervalSubquery()
     TestHelper.assertExpectedObjects(expectedResults, results, "subquery-different-intervals");
   }
 
+  @Test
+  public void testDoubleMeanQuery()
+  {
+    GroupByQuery query = new GroupByQuery.Builder()
+        .setDataSource(QueryRunnerTestHelper.DATA_SOURCE)
+        .setGranularity(Granularities.ALL)
+        .setQuerySegmentSpec(QueryRunnerTestHelper.FIRST_TO_THIRD)
+        .setAggregatorSpecs(
+            new DoubleMeanAggregatorFactory("meanOnDouble", "index")
```

Comment:
I think if using column `doubleNumericNull` instead of `index` you would get better coverage on the null values

Reply:
updated. A similar test case is also added in DoubleMeanAggregationTest